sites/game-client/client.ejs

```diff
@@ -35,13 +35,16 @@ absolute URL.
             <p id="error-message"></p>
           </div>
           <p id="error-handler-message"></p>
-        </div> 
+        </div>
         <% inDiv.forEach((i) => { %>
           <%- include(i); %>
         <% }); %>
       </div>
       <div id="notification-bar">
```

Comment:
Not really sure if we'll have a notification "bar" with flyout notification. We'll revisit this when the design is out.